test/conftest.py

```diff
@@ -48,3 +55,62 @@ def test_session(test_engine):
         yield sessionmaker(test_engine)
     finally:
         close_all_sessions()
```

Comment:
The thinking was that these fixtures could be useful for other harvesting tests.